lib/primer/view_components/linters/view_component_html_classes_counter.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require_relative "base_linter"
+
+# Load all the other linters so we can filter out their restricted
+# CLASSES—they will be responsible for complaining about the use of
+# those HTML classes.
+Dir[File.join(__dir__, "*.rb")].sort.each do |file|
+  require file unless file == __FILE__
+end
+
+module ERBLint
+  module Linters
+    # Counts the number of times a class reserved for ViewComponents is used
+    class ViewComponentHtmlClassesCounter < BaseLinter
```

Comment:
One suggestion here, I think we could have a better name like `DisallowComponentCssCounter`